addons/website/static/src/snippets/s_alert/options.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module **/
+
+import options from "@web_editor/js/editor/snippets.options";
+import fonts from '@web_editor/js/wysiwyg/fonts';
+
+options.registry.Alert = options.Class.extend({
+
```

Comment:
My linter tells me to remove this blank line...

Reply:
don't trust him 😸
There is always empty lignes before and after this type of comment section.
![image](https://github.com/user-attachments/assets/9c16637d-1d3f-4adb-a240-e10c553699fe)